Ontology/Asset/Equipment/HVACEquipment/TerminalUnit/FanCoilUnit/FanCoilUnit.json

```diff
@@ -24,6 +24,32 @@
       },
       "writable": true
     },
+    {
+      "@type": "Property",
+      "displayName": {
+        "en": "mountingType"
+      },
+      "name": "mounting type",
```

Comment:
Will change this to "mounting", to use same nomenclature as for other classes

Reply:
Agreed